hawc/apps/assessment/migrations/0030_assessment_epi_version.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.10 on 2022-01-03 20:11
+
+from django.db import migrations, models
+
+
+def set_epi_version(apps, schema_editor):
+    # set epi_version to 1 on existing assessments
+    Assessment = apps.get_model("assessment", "Assessment")
+    db_alias = schema_editor.connection.alias
```

Comment:
remove the db_alias command? I think based on other migrations in hawc, you can just do

```python
apps.get_model("assessment", "Assessment").objects.update(epi_version=1)
```

Reply:
One more update, let's try to only update those where we have epi data extracted. 

```python
from django.db models import Count
Assessment.objects.annotate(n_sp=Count("references__study__study_populations")).filter(n_sp__gt=0).count()
```

There should be around 40?